install_config/configuring_gce.adoc

```diff
@@ -18,6 +18,23 @@ volumes as persistent storage] for application data. After GCE is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+[[configuring-azure-permissions]]
+== Permissions
+Configuring GCE for {product-title} requires following role:
+
+* *roles/owner* - To create service accounts, cloud storage, instances, images, templates, Cloud
```

Comment:
These are very generic permissions, do we have anything more security restrictive?